website/screens/components/wizard/code/examples/controlled.ts

```diff
@@ -0,0 +1,50 @@
+import { DxcWizard, DxcInset } from "@dxc-technology/halstack-react";
+import { useState } from "react";
+
+const code = `() => {
+    const [myCurrentStep, setMyCurrentStep] = useState(2);
+    const onStepClick = i => {
+        setMyCurrentStep(i);
+    };
+
+    const userIcon = (
```

Comment:
what do you think about moving this icon to an external file to not have so much code in the example?